Exercise_A/index.html

```diff
@@ -6,40 +6,242 @@
+              <p >Bullshit</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    
+    <div class="bg-dark">
+
+      <div class="container py-4">
+        <div class="row text-center justify-content-center">
+          <div class="col ">
+            <h3 class="display-6 text-white py-4">Seriously, It's This Fucking Template!</h3>
+            <div class="">
+
+              <a class=" wow animate__tada px-4 py-3 my-3 fw-bold btn btn-light rounded-pill" href="#" data-wow-duration="1s">THAT'S AMAZING!</a></button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
     <!-- contact me container -->
+<div class="container my-5  " id="call-me">
+  <div class="row text-center justify-content-center  py-3">
+    <div class="col col-lg-8 col-12 ">
+      <h1>My Contact Information</h1>
+      <p>Here's my phone number and email address, because hey that's part of the template and why would you not include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in from all the companies wanting my insane website design skills.</p>
+      <br>
+      <p>Honestly this template does look really nice, though</p>
+    </div>
+    <div class="row text-center justify-content-center py-3">
 
+      <div class="col col-xl-4  col-12">
+        <i class="fas fa-phone-alt fa-2x wow animate__bounceIn"></i>
+        <p>+1-608-317-1888</p>
+      </div>
+      <div class="col col-xl-4 col-md-8 col-12">
+        <i class="far fa-envelope fa-2x wow animate__bounceIn"></i>
+        <a class="d-block color " href="mailto:tab@report.ly">holy@shit.com</a>
+      </div>
+    </div>
+  </div>
+</div>
     <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
+    <script
+    src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+    crossorigin="anonymous"
+  ></script>
+
+  <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-U1DAWAznBHeqEIlVSCgzq+c9gqGAJn5c/t99JyeKa9xxaYpSvHU5awsuZVVFIhvj"
+      crossorigin="anonymous"
+    ></script>
 
+  <script
+    src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
+    integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
+    crossorigin="anonymous"
+  ></script>
+<script src="https://kit.fontawesome.com/3248d77ee6.js" crossorigin="anonymous"></script>
     <!-- 
       All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
      -->
+     <script src="./assets/js/script.js"></script>
+     <script src="./assets/js/wow.js"></script>
+              <script>
+              new WOW().init();
+              </script>
+
   </body>
 </html>

```

Comment:
Bravo! You have nailed the cloning of the website. It shows that you have excellent knowledge of HTML, CSS, Bootstrap, DOM, and JavaScript. Keep up the good work, and keep on grinding.